doc/resources.md

```diff
@@ -29,9 +29,15 @@ A secondary domain is `nix.dev`, which hosts [the main documentation](https://ni
 See the [infra repository](https://github.com/nixos/infra) for more information.
 To request permissions or changes to the infrastructure, get in touch with the [infrastructure team](https://nixos.org/community/teams/infrastructure/).
 
+[NixCon](./nixcon.md) has a separate domain – `nixos.org` – with subdomains dedicated to
```

Comment:
You probably meant nixcon.org not nixos.org there, right?

Reply:
And the nixcon.org domain is also managed by the infra team.

Reply:
Ugh yes.

Reply:
#112